### PR TITLE
preact10 support

### PR DESCRIPTION
**Pull request by aotarola**
Small audit fix and preact10 support :)

**BREAKING CHANGE**: Forces preact10

---
_This PR was started by: [git wf pr](https://github.com/groupon/git-workflow/releases/tag/v1.0.3)_

### CODE REVIEW

File: lib/phy.js [hunk 1]
```diff
@@ -38,6 +38,7 @@ const preact = require('preact');
 const objectCtorString = Object.toString();
 function isPlainObject(obj) {
   return (
+    !obj.hasOwnProperty('props') &&
```
Comment (aotarola) [inline lib/phy.js]: On `preact` < 10, the `h` function returned a [`VNode` function](https://github.com/preactjs/preact/blob/v8.5.0/src/h.js#L76), now in `preact` 10, it returns a [POJO](https://github.com/preactjs/preact/blob/master/src/index.d.ts#L13-L30). I'm not 100% convinced that this is the best way to identify the new `VNode` Object, maybe I can add more conditions?

Thoughts?

Comment (dbushong) [reply lib/phy.js]: We use this to determine if the 2nd argument is an attrs hash/dictionary/POJO or the first kid in a potential varargs list of kids, which could be: a string (no danger of confusion), an Element, or an array of kids (handled by line 42/43).

So really we need to know that these `attrs` you passed us aren't an `Element`... and that's the thing you linked to.  OK, I'm on the same page as you now.

I'm a bit concerned that someone might happen to want to pass an attribute named `props` to something though - unless that's already forbidden, in which case we're fine.

Comment (aotarola) [reply lib/phy.js]: How about detecting the presence of  `type`, `props`, `key`, and `ref` attributes? since they will *always* be in a `VNode` element.

Comment (dbushong) [reply lib/phy.js]: That will probably make it a "good enough" heuristic.  We should rename the function `isAttributes` at this point though.

File: lib/phy.js [hunk 2]
```diff
@@ -34,12 +34,16 @@
 
 const preact = require('preact');
 
+// VNode props https://github.com/preactjs/preact/blob/master/src/index.d.ts#L14-L17
+const VNODE_ATTRS = ['props', 'type', 'key', 'ref'];
+
 // simplified from lodash
-const objectCtorString = Object.toString();
-function isPlainObject(obj) {
+const objectCtoString = Object.toString();
```
Comment (dbushong) [inline lib/phy.js]: the intent of this function name was an abbreviation for `objectConstructorString` (hence `Ctor`) - not the clearest name, maybe `objectConstructorToString` or `objectCtorToString` or `objectToString`?

Comment (aotarola) [reply lib/phy.js]: `objectToString` sounds better :)

**Pull request merged**